src/game/table.rs

```diff
@@ -0,0 +1,53 @@
+#![allow(dead_code)]
+
+use super::card::{self, Monad, Card, Deck};
+use std::iter::repeat_with;
+
+pub struct Table {
+    pub discard: Deck,
+    pub common: Deck,
+    pub bi: Deck,
+    pub tri: Deck,
+    pub quad: Deck,
+    pub quint: Deck,
+    pub monad: Vec<Monad>,
```

Comment:
If you're going to align the instantiations of the struct, you should also align the definition!

Reply:
I whole-heartedly agree!